setup.py

```diff
@@ -71,7 +71,7 @@ class CustomDist(sdist):
     def run(self):
         copy_tree(f'src/main/proto/{package_name}', package_name)
 
-        copy_tree(f'src/gen/main/python/{package_name}', package_name)
+        copy_tree(f'src/gen/main/services/python/{package_name}', package_name)
```

Comment:
Shouldn't we change the path on line 41?

Reply:
I add options `pythonPath` to our service generator and all generated python service's file will be in this path. I think, this command copy service's files.

Reply:
Yes, but if we look at line 41 we will see a very similar path. I am not sure but it seems to me that we should also change it.
@RaevskiySV Could you please help us with that? Should we correct the path on line 41 as well?

Reply:
Code on line 41 is responsible for path where generated code from proto files (not services) will be stored.
If we store our services under new directory, we should add another line of copying.
Additionally, we should add `__init__.py` file in order to directory become a Python package.